docs/application/dotnet/guides/nui/nui-components/Toast.md

```diff
@@ -7,6 +7,16 @@ You can use a toast to provide simple messages when the user does not need to ma
 
 ![Toast](./media/toast.png)
 
+> [!NOTE]
+> Toast is deprecated in API8 (Tizen 6.0) and will be removed in API10.

```

Comment:
Please rephrase to:

Toast is deprecated since Tizen 6.0 and will be removed after two releases.